HousingRepairsOnlineApi/UseCases/RetrieveAvailableAppointmentsUseCase.cs

```diff
@@ -18,16 +20,32 @@ public RetrieveAvailableAppointmentsUseCase(IAppointmentsGateway appointmentsGat
             this.sorEngine = sorEngine;
         }
 
-        public async Task<IEnumerable<Appointment>> Execute(string repairLocation, string repairProblem, string repairIssue, string uprn)
+        public async Task<List<ApplicationTime>> Execute(string repairLocation, string repairProblem,
+            string repairIssue, string uprn)
         {
             Guard.Against.NullOrWhiteSpace(repairLocation, nameof(repairLocation));
             Guard.Against.NullOrWhiteSpace(repairProblem, nameof(repairProblem));
             Guard.Against.NullOrWhiteSpace(repairIssue, nameof(repairIssue));
             Guard.Against.NullOrWhiteSpace(uprn, nameof(uprn));
             var repairCode = sorEngine.MapSorCode(repairLocation, repairProblem, repairIssue);
+
+            var convertedResults = new List<ApplicationTime>();
+
             var result = await appointmentsGateway.GetAvailableAppointments(repairCode, uprn);
+            convertedResults.AddRange(result.Select(ConvertToHactAppointment));
```

Comment:
This and line 32 could be consolidated to:

`var convertedResults = result.Select(ConvertToHactAppointment).ToList()`

```suggestion
            var convertedResults = result.Select(ConvertToHactAppointment).ToList()
```
If this suggestion is accepted, the above (line 32) suggestion in this file would also need to be accepted.